telecine/build.gradle

```diff
@@ -6,7 +6,7 @@ def versionBuild = 0 // bump for dogfood builds, public betas, etc.
 
 buildscript {
   dependencies {
-    classpath 'com.android.tools.build:gradle:2.2.3'
+    classpath 'com.android.tools.build:gradle:3.0.0-alpha2'
```

Comment:
Some people got sad and complained when they were stuck on an alpha for months. Xav said it's function is relatively diminished now anyway, though, as it was just meant to keep people from being stuck on old 0.x and 1.x alphas years ago.